setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+setup(
+    author="CS50",
+    author_email="sysadmins@cs50.harvard.edu",
+    classifiers=[
+        "Intended Audience :: Developers",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Topic :: Software Development :: Libraries :: Python Modules"
+    ],
+    description="CS50 library for Python",
+    install_requires=["SQLAlchemy"],
+    keywords="cs50",
+    name="cs50",
+    packages=["cs50"],
+    url="https://github.com/cs50/python-cs50",
+    version="1.3.0"
```

Comment:
Interesting. Is calling the var `__version__` conventional? This would seem to mean that `setup.py` needs to be installed along with, e.g., `cs50.py` itself. Is that indeed the norm with PyPi packages, does `setup.py` end up getting installed somewhere? 

Separately, what's to be the relationship between the version number in `setup.py` and our GitHub repo's branches? Should we automatically (forcibly) create a GitHub tag and release called `v#.#.#` anytime there's a push, extracting the release's version number from `setup.py` too?